arangod/RestServer/ServerFeature.cpp

```diff
@@ -137,53 +117,12 @@ another mode.)");
 }
 
 void ServerFeature::validateOptions(std::shared_ptr<ProgramOptions> options) {
-  int count = 0;
-
-  if (_console) {
-    _operationMode = OperationMode::MODE_CONSOLE;
-    ++count;
-  }
-
-  if (!_scripts.empty()) {
-    _operationMode = OperationMode::MODE_SCRIPT;
-    ++count;
-  }
-
-  if (1 < count) {
-    LOG_TOPIC("353cd", FATAL, arangodb::Logger::FIXME)
-        << "cannot combine '--console', '--javascript.unit-tests' and "
-        << "'--javascript.script'";
-    FATAL_ERROR_EXIT();
-  }
-
   DatabaseFeature& db = server().getFeature<DatabaseFeature>();
 
-  if (_operationMode == OperationMode::MODE_SERVER && !_restServer &&
-      !db.upgrade() &&
+  if (!_restServer && !db.upgrade() &&
       !options->processingResult().touched("rocksdb.verify-sst")) {
     LOG_TOPIC("8daab", FATAL, arangodb::Logger::FIXME)
-        << "need at least '--console', '--javascript.unit-tests' or"
-        << "'--javascript.script if rest-server is disabled";
-    FATAL_ERROR_EXIT();
-  }
-
-  bool supportsV8 = false;
-#ifdef USE_V8
-  V8DealerFeature& v8dealer = server().getFeature<V8DealerFeature>();
-
-  if (v8dealer.isEnabled()) {
-    if (_operationMode == OperationMode::MODE_SCRIPT) {
-      v8dealer.setMinimumExecutors(2);
-    } else {
-      v8dealer.setMinimumExecutors(1);
-    }
-    supportsV8 = true;
-  }
-#endif
-  if (!supportsV8 && _operationMode != OperationMode::MODE_SERVER) {
-    LOG_TOPIC("a114b", FATAL, arangodb::Logger::FIXME)
-        << "Options '--console', '--javascript.unit-tests'"
-        << " or '--javascript.script' are not supported without V8";
+        << "Rest server cannot be disabled any more, because it is pointless.";
```

Comment:
Should we just remove this? If not we should probably improve the error message.